src/app/login/page.tsx

```diff
@@ -10,10 +10,13 @@ export default function LoginPage() {
           <CardHeader>
             <h2 className="text-xl font-bold">Sign In</h2>
           </CardHeader>
-          <CardContent>
+          <CardContent className="grid gap-4">
             <Link href="/login/github">
               <Button>Continue with Github</Button>
             </Link>
+            <Link href="/login/google">
+              <Button>Continue with Google</Button>
+            </Link>

```

Comment:
In the case of accessibility, it is better to do:
```suggestion
	    <Button asChild>
              <Link href="/login/google">
                Continue with Google
              </Link>
            </Button>
```

Reply:
I see thanks for the suggestion!